person.rb

```diff
@@ -3,17 +3,18 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
### [ Required Changes ]

It would be nice if you could add a method for adding rentals for a person to achieve the 'has-many' association. It's required you have add_rental method in this class. Kindly work on that 👍🏽


### [ Optional Changes ]

For a getter and setter, you could use the attr_accessor as it does bother, getting and setting a value.

`attr_accessor :name, :age, :rentals`